mod-pubsub-client/src/main/java/org/folio/util/pubsub/PubSubClientUtils.java

```diff
@@ -169,6 +174,47 @@ private PubSubClientUtils() {
     return publishersResult;
   }
 
+  /**
+   * Util method to unregister external module in PubSub.
+   *
+   * @param params - okapi connection params
+   * @return future with true if module was unregistered successfully
+   */
+  public static CompletableFuture<Boolean> unregisterModule(OkapiConnectionParams params) {
```

Comment:
@RuslanLavrov, do you think it is worth having separate util methods for unregistering as subscriber and as publisher?

Reply:
@KaterynaSenchenko, according to the description, there is necessary to unregister a module in pubsub when the module is disabled for a tenant. So, as I understand all information about module publications and subscriptions should be deleted after module disabling. @KaterynaSenchenko, @OleksiiKuzminov, @VRohach, let me know please if you think that it is worth to add them. I will do it.